README.md

```diff
@@ -16,14 +16,14 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 # System Requirements
 
 ## Minimum
-* OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
+* OS: Windows 7 SP1 or newer or GNU/Linux (32-bit or 64-bit)
 * CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up) @ 1600 STR or better
 * GPU: DirectX 10 GPU or better
 * RAM: 2GB or more
 
 ## Recommended
-* OS: Windows 7/8/8.1/10 (64-bit) or GNU/Linux (64-bit)
-* CPU: Intel Haswell (or AMD equivalent) @ 2000 STR or better
+* OS: Windows 10 (64-bit) or GNU/Linux (64-bit)
+* CPU: Any that supports AVX2 (Haswell and up, Ryzen and up) @ 2000 STR or better
```

Comment:
Honestly not all end users might be familiar of CPU architectures, we need something more simple for users to catch I feel. Also might be nice to hyperlink to hyper link the STR value to the STR description sectio.

Reply:
Something like Core I7 4770K and Ryzen 5 2600?

Reply:
Just the STR is fine, no need to mention models unless you absolutely have to (ex avx2 support).